parts/configure-swarmmode-cluster.sh

```diff
@@ -154,7 +164,7 @@ updateDockerDaemonOptions()
     # also have it bind to the unix socket at /var/run/docker.sock
     sudo bash -c 'echo "[Service]
     ExecStart=
-    ExecStart=/usr/bin/docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock
+    ExecStart=/usr/bin/docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock -H fd:// --experimental=true --metrics-addr=0.0.0.0:4999

```

Comment:
@kbhattmsft, can you clarify what you mean by?: "changes suggested above not complete and don't work".

Above was one suggestion. Another one what I shared in the email to make it easier and straightforward.

// SwarmModeConfig array
type SwarmModeConfig struct {
    OrchestratorOptions map[string]string `json:"orchestratorOptions,omitempty"`
}

What is the blocking you?